modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/OrchestratorService.java

```diff
@@ -68,7 +72,9 @@ public void onApplicationEvent(ApplicationEvent event) {
      * @param oclLocation the location of the OCL descriptor.
      * @throws Exception if registration fails.
      */
+    @Async("taskExecutor")
```

Comment:
Can you also please check if the API now returns HTTP 202 immediatly and swagger UI also shows response code 202?

Reply:
It returns HTTP 200 immediatly when call the API

Reply:
The start and stop API using @Async now return HTTP 202 immediatly